locksmith/config/config.js

```diff
@@ -1,4 +1,11 @@
-require('dotenv').config()
+const path = require('path')
+const dotenv = require('dotenv')
+
+const unlockEnv = process.env.UNLOCK_ENV || 'dev'
+
+dotenv.config({
+  path: path.resolve(__dirname, '..', `.env.${unlockEnv}.local`),
```

Comment:
In what context would someone be using `env.staging.local` or `env.production.local` locally?